frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -281,7 +289,33 @@ export class Query extends Component {
           type: 1,
         },
       });
+    } else if (this.state.file === false && emptyFullName !== false) {
+      emptyFullName += 1;
+      this.setState({
+        message: {
+          message:
+            'FullName is required, it is missing at row ' + emptyFullName,
+          update: !this.state.message.update,
+          trigger: true,
+          type: 1,
+        },
+      });
+    } else if (this.state.file === false && emptyUsernames !== false) {
+      emptyUsernames = emptyUsernames + 1;
+      this.setState({
+        message: {
+          message:
+            'Fill either gerrit or phabricator username at row ' +
```

Comment:
Wording: "Both Gerrit and Phabricator fields cannot be left blank. Provide username for one of these accounts in row +  .... !"